src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,145 @@
+package duke.parser;
+
+import duke.command.*;
+import duke.exception.*;
+import duke.storage.Storage;
+import duke.task.*;
```

Comment:
it is best to not use the * when importing, what do you think?

Reply:
oh shit i think IntelliJ refractor code function auto combined all my imports into * again... thanks for the catch!